test/scripts/parseJson.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import BpmnXmlParser from '../../src/component/parser/xml/BpmnXmlParser';
+import { readFileSync } from '../helpers/file-helper';
+import * as path from 'path';
+import { defaultBpmnJsonParser } from '../../src/component/parser/json/BpmnJsonParser';
+import { BpmnJsonModel } from '../../src/component/parser/xml/bpmn-json-model/BPMN20';
+import BpmnModel from '../../src/model/bpmn/BpmnModel';
+import clipboardy from 'clipboardy';
+
+const __dirname = path.resolve();
+const myArgs = process.argv.slice(2);
```

Comment:
IMHO, it is strange to pass the type of export without an option name, this doesn't make the command clear enough
I suggest to use a parsing utility, for instance `minimist` (see #645)

This would allow us to use a command like in the following
```
node ./..../utils.mjs test/fixtures/bpmn/non-regression/gateways.bpmn --output json
```

```
node ./..../utils.mjs  --output model test/fixtures/bpmn/non-regression/gateways.bpmn
```

Reply:
this is cool  -  minimist added